src/middleware/builtin.ts

```diff
@@ -338,10 +354,11 @@ type CallbackIdentifiedBody =
   | InteractiveMessage
   | DialogSubmitAction
   | MessageAction
```

Comment:
Perhaps we should update `MessageAction` -> `MessageShortcut` throughout the codebase, but that can be a TODO for later